boards/FSM/Inc/Io/Io_Brake.h

```diff
@@ -7,3 +7,43 @@
  * @return True if the brake is actuated, false if the brake is not actuated
  */
 bool Io_Brake_IsActuated(void);
+
+/**
+ * Check if the MSP-300-2K5-P-3-N-1 is in an open circuit or short circuit
+ * state.
+ * @return True if the sensor is open or short circuited, false if the sensor is
+ * not
+ */
+bool Io_MSP3002K5P3N1_IsOpenOrShortCircuit(void);
+
+/**
+ * Get the pressure in psi from the MSP-300-2K5-P-3-N-1 brake pressure sensor
+ * @return The pressure of the primary brake pressure sensor in psi
+ */
+float Io_MSP3002K5P3N1_GetPressurePsi(void);
+
+// TODO Update descriptions
+/**
+ * Check if the [REAR BRAKE PRESSURE SENSOR] is in an open circuit or short circuit
+ * state.
+ * @return True if the sensor is open or short circuited, false if the sensor is
+ * not
+ */
+bool Io_RearBrake_IsOpenOrShortCircuit(void);
```

Comment:
Convention is these are all prefixed `Io_Brake`
So smtg like `Io_Brake_IsRearBrakeOcOrSc`

Reply:
Fixed in 3410f90f08b5f9eaf133ec936fc996b2448f2eba